docs/source/xrpl.models.rst

```diff
@@ -1,6 +1,9 @@
 XRPL Models
 ===========
 
+Use these classes to validate the formats for all types of data coming from or
+going to the XRP Ledger.
+
```

Comment:
Not just validation, but also contraction of requests

Reply:
What do you mean by "contraction"?